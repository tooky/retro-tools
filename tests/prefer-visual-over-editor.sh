VISUAL=fake-visual EDITOR=fake-editor retro_date=1997-06-05 retro add
