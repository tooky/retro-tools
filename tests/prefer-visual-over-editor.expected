VISUAL=fake-visual EDITOR=fake-editor retro_date=1997-06-05 retro add
VISUAL doc/retro/1997/06/05.md
doc/retro/1997/06/05.md
